### PR TITLE
Added the checkout page.

### PR DESCRIPTION
**Pull request by ionutvmi**
Added the checkout page.
Added the checkout page.
Added the checkout page.

### CODE REVIEW

File: web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ro.tefacprogramator.m4;
+
+public class CheckoutController {
+    public void showCheckoutPage() {
```
Comment (ionutvmi) [inline web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java:4]: test comment

Comment (CipDanila) [reply web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java:4]: reply. e bine?

Comment (ionutvmi) [reply web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java:4]: e bine

File: web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package ro.tefacprogramator.m4;
+
+public class CheckoutController {
```
Comment (CipDanila) [inline web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java:3]: comment

Comment (rodica-m34) [reply web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java:3]: comment

File: web-api/src/main/java/ro/tefacprogramator/m4/App.java [hunk 3]
```diff
@@ -17,5 +17,7 @@ public static void main( String[] args )
 
         HomepageController hp = new HomepageController();
 
+        CheckoutController c1 = new CheckoutController();
+
```
Comment (CipDanila) [inline web-api/src/main/java/ro/tefacprogramator/m4/App.java:21]: checked

**Review by rodica-m34 — COMMENTED** [review]
ok

File: web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java (hunk 2 above)
Comment (mariscostin96) [inline web-api/src/main/java/ro/tefacprogramator/m4/CheckoutController.java:3]: hauhau

**Review by rodica-m34 — APPROVED** [review]

**Review by CipDanila — APPROVED** [review]

**Pull request merged**